### DISCUSSION TITLE
Handling sites without mean flow data

### DISCUSSION THREAD

**Discussion by jds485** [USGS-R/regional-hydrologic-forcings-ml#39] [Q&A] [answered]
Looks like we have only min and max flow data for site 07311600 (stat_cd 00001 and 00002). It seems odd that NWIS would have those and not have the mean. Is there a quick way to check in a web GUI if the mean data are available and not showing up in the API?

**Comment by slevin75** [comment, marked as answer]
I just checked NWIS and it does only have minimum and maximum discharge for some reason.  I have never seen this at a gage before.

**Reply by slevin75** [reply]
![image](https://user-images.githubusercontent.com/18313247/151007969-00941b2b-7cb8-4079-9d42-096e8622c020.png)
